contrib/opencensus-ext-gevent/README.rst

```diff
@@ -0,0 +1,27 @@
+OpenCensus gevent Exporter
+============================================================================
+
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opencensus-ext-gevent.svg
+   :target: https://pypi.org/project/opencensus-ext-gevent/
+
+Installation
+------------
+
+::
+
+    pip install opencensus-ext-gevent
+
+Usage
+-----
+
+The **OpenCensus gevent helper** ...
+
+TBD
```

Comment:
@aberres This is the good place to put the original doc that you prepared in the README.